content/getting-started/sections/installing-atom.md

```diff
@@ -159,6 +159,41 @@ $ sudo zypper in -y atom.x86_64.rpm
 
 {{/linux}}
 
+#### Updating Atom
+
+You should consider updating Atom periodically for the latest improvements to the software. When Atom receives hotfixes for security vulnerabilities you will want to update your version of Atom as soon as possible. This section will explain how to update Atom on your system.
```

Comment:
Reading this out loud, it feel like the first 2 sentences could you use a connector?  E.g.:

```suggestion
You should consider updating Atom periodically for the latest improvements to the software.  Additionally, when Atom receives hotfixes for security vulnerabilities, you will want to update your version of Atom as soon as possible. This section will explain how to update Atom on your system.
```